scripts/linux/install.sh

```diff
@@ -10,6 +10,12 @@ then
     exit -1
 fi
 
+if [ -n "$DSN_ROOT" -a -f "$DSN_ROOT/lib/libdsn.core.so" ]
```

Comment:
I'm wondering whether removing the copy action in build is more appropriate in terms of the semantic? Or, it seems wired as if I cannot overwrite an existing installation (in fact we do during build, but...).